changelog/fragments/1755810145-Reset-trace-headers-on-bulk-items-when-returning-to-pool.yaml

```diff
@@ -0,0 +1,32 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: bug-fix
+
+# Change summary; a 80ish characters long description of the change.
+summary: Reset trace links on bulk items when returning to pool
```

Comment:
Correct me if I'm wrong but this change is related to getting APM traces for Fleet Server.  If so, there is no user impact from this change, right?  In that case, do we even need a CHANGELOG entry?

Reply:
Most likely, but IIRC a self-managed user can enable tracing